### PR TITLE
CIRC-5610 IRONdb shard compactor

### PR DESCRIPTION
**Pull request by esproul**

### CODE REVIEW

File: content/irondb/administration/compacting-numeric-rollups.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  rollup period and the start and end timestamps that they represent. This
+  option may be specified multiple times to compact more than one shard. Shards
+  will be compacted serially.
+
+Run `/opt/circonus/bin/shard_compactor --help` for full usage information. The
+tool must be run as the unprivileged user that IRONdb runs as, typically
+`nobody`.
+
+This is an online operation (the IRONdb service must be running). Each shard
+will be put into an "offline" mode while it is being compacted. Requests for
```
Comment (bdunavant) [inline content/irondb/administration/compacting-numeric-rollups.md:32]: Should we link the "offline API" docs from here?

File: content/irondb/administration/compacting-numeric-rollups.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+data within the shard will be redirected to other cluster nodes during the
+operation.
+
+## Caveats
+
+Compaction should only be performed on shards that are no longer getting new
```
Comment (bdunavant) [inline content/irondb/administration/compacting-numeric-rollups.md:38]: You have them as separate paragraphs, but maybe these should be bullets as well, to drive home that there are 3 separate caveats for those with less attention span

File: content/irondb/administration/compacting-numeric-rollups.md (hunk 1 above)
Comment (esproul) [reply content/irondb/administration/compacting-numeric-rollups.md:32]: I can do that now; they didn't exist when I wrote this.

Comment (esproul) [reply content/irondb/administration/compacting-numeric-rollups.md:32]: Actually, no, those docs haven't integrated yet.

File: content/irondb/administration/compacting-numeric-rollups.md (hunk 2 above)
Comment (esproul) [reply content/irondb/administration/compacting-numeric-rollups.md]: Done.

**Pull request merged**